src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsCC.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import FacilityAddress from '../../../components/FacilityAddress';
+import AppointmentDateTime from '../AppointmentDateTime';
+import Breadcrumbs from '../../../components/Breadcrumbs';
+import CalendarLink from './CalendarLink';
+import TypeHeader from './TypeHeader';
+import PrintLink from './PrintLink';
+import RescheduleOrCancelAlert from './RescheduleOrCancelAlert';
+import ProviderName from './ProviderName';
+import CCInstructions from './CCInstructions';
+import { getTypeOfCareById } from '../../../utils/appointment';
+
+export default function DetailsCC({ appointment, useV2 = false }) {
+  const header = 'Community care';
+  const facility = appointment.communityCareProvider;
+  const typeOfCare = getTypeOfCareById(appointment.vaos.apiData.serviceType);
+
+  const ShowTypeOfCare = () => {
+    return (
+      useV2 &&
+      typeOfCare && (
+        <>
+          <h2
+            className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0"
+            data-cy="community-care-appointment-details-header"
+          >
+            <div className="vads-u-display--inline">Type of care</div>
+          </h2>
+          <div>{typeOfCare?.name}</div>
+        </>
+      )
+    );
+  };
+
+  return (
+    <>
+      <Breadcrumbs>
+        <Link to={`/cc/${appointment.id}`}>Appointment detail</Link>
+      </Breadcrumbs>
+      <h1>
+        <AppointmentDateTime appointment={appointment} />
+      </h1>
+      <ShowTypeOfCare />
+      <TypeHeader isCC>{header}</TypeHeader>
+      <ProviderName appointment={appointment} />
```

Comment:
add the `useV2` parameter otherwise the ProviderName.jsx will use its default value set at "false"
```suggestion
      <ProviderName appointment={appointment} useV2={useV2} />
```